[program:archlinux_gunicorn]
process_name=%(program_name)s
numprocs=1
directory=/home/zeus/archlinux
command=/home/zeus/.virtualenvs/archlinux/bin/python manage.py run_gunicorn -c /home/zeus/archlinux/conf/gunicorn.conf.py
user=zeus
autostart=true
autorestart=true
redirect_stderr=true
stopwaitsecs=60
stopsignar=INT
stderr_logfile=/home/zeus/logs/archlinux_sv_guni-error.log
stdout_logfile=/home/zeus/logs/archlinux_sv_guni-out.log
stdout_logfile_maxbytes=10MB
stdout_logfile_backups=10
stdout_capture_maxbytes=1MB

[group:archlinux]
programs=archlinux_gunicorn